basyx.aasrepository/basyx.aasrepository-feature-mqtt/src/main/java/org/eclipse/digitaltwin/basyx/aasrepository/feature/mqtt/MqttAasRepository.java

```diff
@@ -94,7 +97,7 @@ public void deleteAas(String aasId) {
 
 	@Override
 	public String getName() {
-		return decorated.getName();
+		return aasRepositoryName;
```

Comment:
Is there any reason why you are not using decorated.getName()? This will make the private variable aasRepositoryName in this class obsolete.